package.json

```diff
@@ -22,7 +22,10 @@
     "start": "yarn build && env-cmd node build/index.js",
     "build": "tsc",
     "watch": "nodemon --exec \"yarn start\" --watch src -e ts",
-    "package": "yarn build && pkg -t win-x64 -o dist/impftermin.exe build/index.js --public",
+    "package": "yarn build && pkg -t win-x64 -o dist/impftermin.exe build/index.js --public && pkg -t macos-x64 -o dist/impftermin-macos build/index.js --public && pkg -t linux-x64 -o dist/impftermin-linux build/index.js --public",
```

Comment:
Thanks @quinn-dev ! Was also working on this but can only test on Windows currently..
How about changing this to `yarn build && yarn package:win && yarn package:macos && yarn package:linux`?

Reply:
That was my initial thought as well, but then you have to rebuild `src/index.js` at least three times in a row (since every `yarn package:os` command causes a rebuild. The current solution requires six rebuilds.

Using my ugly solution (which only requires one rebuild), building the packages takes roughly 21 seconds. Using your prettier and refactored solution, the same task takes about 49 seconds.

The prettiest solution would be to create a `pkg` section in `package.json`:
```json
  "pkg": {
    "targets": [ "win-x64", "macos-x64", "linux-x64" ],
    "outputPath": "dist",
    "public": true
  }
```
But then we couldn't choose the binaries' names, which would default to `impftermin-win.exe`, `impftermin-macos` and `impftermin-linux`. If these names are acceptable, this would undoubtedly be the best solution.

Otherwise we can either use the ugly solution I initially proposed or your prettier but more ineffective solution.

Reply:
The `pkg` section in `package.json` doesn't seem to be fully functional (the `public` argument doesn't work).

I'd suggest we keep your solution but remove the initial `yarn build` from the entry since it already runs before every `pkg` command.

Reply:
I implemented this change in #12.